### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mukeshkharita**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+	
+	Run the project ResultApp in Android Studio.
+	
+	Internet should be connected for installing the libraries.
+	
+	Libraries will automatically install.
+	
+Database in your laptop:
+
+	mysql should be install
```
Comment (Akarshit) [inline README.md]: Please make it "should be installed"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+	
+		https://www.digitalocean.com/community/tutorials/how-to-install-mysql-on-ubuntu-14-04
+		
+	For dump the sql in own PC
```
Comment (Akarshit) [inline README.md]: Please make it "To dump the.."

**Comment by mukeshkharita** [comment]
Done

**Comment by Akarshit** [comment]
@mukeshkharita It would be awesome if you could squash the two commits into one.

**Comment by mukeshkharita** [comment]
Sir How

**Comment by Akarshit** [comment]
@mukeshkharita try [this](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) link.

No need for "sir', just use the nick.

**Comment by mukeshkharita** [comment]
But this all done at github so how i can.

**Comment by mukeshkharita** [comment]
Is it right?
screenshot from 2016-05-19 16-14-32

**Comment by Akarshit** [comment]
@mukeshkharita Just squash all those commits into one in the same way as described in the link above.

**Comment by Akarshit** [comment]
@mukeshkharita We decided to change the directory structure a little bit. The README has been updated also.

**Pull request closed without merging**